configs/projects/puppet-agent.rb

```diff
@@ -153,7 +153,6 @@
   proj.license "See components"
   proj.vendor "Puppet Labs <info@puppetlabs.com>"
   proj.homepage "https://www.puppetlabs.com"
-  proj.target_repo "PC1"
```

Comment:
We no longer need to set this in vanagon. We don't care how vanagon stages packages anymore, since we always reorder them with packaging